reference/7.0/Microsoft.PowerShell.Core/About/about_Operators.md

```diff
@@ -554,6 +556,71 @@ B
 A
 ```
 
+If you assign a character range to a string, it's treated the same assigning a
+character array to a string.
+
+```powershell
+PS> [string]$s = 'a'..'e'
+$s
+a b c d e
+$a = 'a', 'b', 'c', 'd', 'e'
+$a
+a b c d e
+```
+
+The characters in the array are joined into a string. The characters are
+separated by the value of the `$OFS` preference variable. For more information,
+see [about_Preference_Variables](about_Preference_Variables.md#ofs).
+
+The order of the characters in the array is determined by the ASCII value of
+the character. For example, the ASCII values of `c` and `X` are 99 and 88,
```

Comment:
Is it worth linking to [this page for the character codes][01] or mentioning the usability of the [GetBytes][02] method to retrieve the value of a given character, like:

```powershell
[System.Text.ASCIIEncoding]::ASCII.GetBytes(5)
[System.Text.ASCIIEncoding]::ASCII.GetBytes(f)
```

```output
53
102
```

[01]: https://learn.microsoft.com/office/vba/language/reference/user-interface-help/character-set-0127
[02]: https://learn.microsoft.com/dotnet/api/system.text.asciiencoding.getbytes

Reply:
Don't want to complicate things. Plus the simpler form I was using to test was `[byte]([char]'X')`.